sdk/eventgrid/azure-eventgrid/azure/eventgrid/_operations/_patch.py

```diff
@@ -99,10 +137,48 @@ def publish_cloud_events(
         :rtype: None
         :raises ~azure.core.exceptions.HttpResponseError:
         """
+    
+    @overload
+    def publish_cloud_events(
+        self,
+        topic_name: str,
+        body: List[Dict[str, Any]],
+        *,
+        binary_mode: Optional[bool] = False,
+        content_type: str = "application/cloudevents-batch+json; charset=utf-8",
+        **kwargs: Any
+    ) -> None:
+        """Publish Single Cloud Event to namespace topic. In case of success, the server responds with an
+        HTTP 200 status code with an empty JSON object in response. Otherwise, the server can return
+        various error codes. For example, 401: which indicates authorization failure, 403: which
+        indicates quota exceeded or message is too large, 410: which indicates that specific topic is
+        not found, 400: for bad request, and 500: for internal server error.
```

Comment:
non-blocking comment: Are these error codes returned in an object or is an error raised? If raised, you may want to move the specific error code information to the :raises: section in the docstring

Reply:
this is what gets generated from the typespec doc they are raised though -- so might require a tsp update then